.dockerignore

```diff
@@ -1,9 +1,16 @@
-.vagrant/
-_build/
-apps/aecuckoo/c_src
-apps/aecuckoo/priv/bin
-apps/aecuckoo/priv/lib
-apps/aecore/priv/keys
-system_test
-erl_crash.dump
-aevm_external
+*
+!rebar.config
+!rebar.lock
+!Makefile
+!rebar3
+!rebar.config.script
+!VERSION
+!apps
+!.git
+!otp_patches
+!test
+!config
+!hooks
+!data
+!docs
+!docker
```

Comment:
Can you please remind me why this needed?

Reply:
it's only `docker/entrypoint.sh` I think

Reply:
correct, whitelisted only entrypoint